chains/zircuit/addresses.yaml

```diff
@@ -0,0 +1,11 @@
+staticMerkleRootMultisigIsmFactory: "0x7008bfED551C7fcC1B168465CD460924A928f4bB"
```

Comment:
These will need to be ordered alphabetically. If you run `yarn lint:fix` it will take care of it for you